packages/server/src/agent/websockets.test.ts

```diff
@@ -21,7 +21,7 @@ describe('Agent WebSockets', () => {
   beforeAll(async () => {
     config = await loadTestConfig();
     config.vmContextBotsEnabled = true;
-    config.heartbeatMilliseconds = 500;
+    config.heartbeatMilliseconds = 5000;
```

Comment:
Why does this need to be increased? Test was flaky?

Reply:
Yeah ..  the heartbeat would come in before the expected "transmit" message.  Ideally there would be a way to say "expect these messages in any order", but didn't see that 

Alternatively, the test file could be refactored to support enabling/disabling the heartbeat as necessary.